### PR TITLE
Enhancement: Add tests, run them on Travis

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] requires `phpunit/phpunit` (and removes the `minimum-stability` flag)
* [x] adds a configuration for `phpunit`
* [x] adds coverage
* [x] adds a configuration for Travis (again)
* [x] adds a build status badge to `README.md` (and removes the broken badge)

:information_desk_person: This only requires to enable the build on Travis again.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 7
+  - hhvm
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```
Comment (localheinz) [inline .travis.yml]: @pleckey 

:information_desk_person: To really make use of caching, one should create a GitHub token and use it to authenticate with the GitHub API:

1. Acquire a GitHub authentication token (go to https://github.com/settings/tokens/new)
2. Install the `travis` command line tool (see https://github.com/travis-ci/travis.rb#installation)
3. Encrypt the GitHub authentication token

    ```
$ travis encrypt GITHUB_TOKEN=foobarbaz
    ```
4. Add the encrypted token to `.travis.yml`

    ```yml
    env:
      global:
        - secure: "..."
    ```
5. Use token to authenticate against GitHub API before installing dependencies

    ```yml
    before_install:  
      - composer config github-oauth.github.com $GITHUB_TOKEN
    ```

**Comment by pleckey** [comment]
@localheinz I will set this up tomorrow, booked for the rest of today - it is appreciated, thank you!

**Comment by localheinz** [comment]
@pleckey 

You're much welcome!

:sun: Have a nice day!

**Comment by pleckey** [comment]
@localheinz just noticed php 5.6 missing from travis run config, please add.  :smile:

**Comment by localheinz** [comment]
@pleckey 

Good catch!

**Comment by pleckey** [comment]
:+1:

**Pull request merged**